sdl-snowflake/src/main/scala/io/smartdatalake/workflow/dataframe/snowflake/SnowparkSubFeed.scala

```diff
@@ -231,6 +253,14 @@ object SnowparkSubFeed extends DataFrameSubFeedCompanion with SmartDataLakeLogge
     }
   }
 
+  override def hash(column: GenericColumn): GenericColumn = {
+    val udfRaiseError = functions.udf((msg: String) => throw new RuntimeException(msg)) // Spark raise_error functions also creates a RuntimeException
```

Comment:
are we missing some logic here? the error is never used

Reply:
Yes, line can be removed.